include/pokemon.h

```diff
@@ -407,6 +407,11 @@ struct SpindaSpot
     u16 image[16];
 };
 
+struct __attribute__((packed)) LevelUpMove {
+    u16 move:9;
+    u16 level:7;
+};
+
```

Comment:
Yeah, it currently doesn't match at point of use.  I haven't been able to fix that.  In the event that someone else can, I leave them this struct.